bokehjs/src/coffee/main.coffee

```diff
@@ -16,7 +16,10 @@ if not window.Float64Array
 
 # binding the libs that bokeh uses so others can reference them
 Bokeh._                 = require("underscore")
-Bokeh.$ = $ = jQuery    = require("jquery")
+if $ || jQuery
+    Bokeh.$ = $ = jQuery = require("jquery").noConflict(true)
+else
+    Bokeh.$ = $ = jQuery = require("jquery")
```

Comment:
I found only one place where `$` is used, but not preceded by `$ = require('jquery')` (`common/geojs_plot`, but there is an explicit note about this, thought not explaining why this has to work this way).